### ISSUE TITLE
So close to what I need

### ISSUE THREAD

**Issue by ggb667** [mojotech/pullr#65]
I tried to hack on this and make it support my local corporate repos, but no dice.

`program
  .version(package.version)
  .option('-g, --github_location <url>', 'url of github')
  ...`

`return {
    github_location : github_location,
   ...`

'var url = options.github_location + '/repos/' + options.intoOwner + '/' + options.intoRepo + '/pulls',
      repo = options.intoRepo,
      head = options.fromOwner + ':' + options.fromBranch,
      base = options.intoBranch;'

But no dice.  Am I missing something?

Tried calling it like:
c:\wk\ms-attraction-reservations>"C:\Program Files\nodejs\node.exe" c:\wk\pullr -g https://mygituub.net -n -l -t "Sonar Jenkins Update" -d "Update Jenkins file so it supports develop branch as opposed to all branches together" -I "develop" -f origin/feature/US32497-1_Sonar -p --plaintext

**Comment by cpjolicoeur** [comment]
@ggb667 Can you provide any output/errors you are getting when you try to run that command?

Does your internal git host follow the same URL structure as GitHub (is it a hosted Github Enterprise instance)?

**Comment by ggb667** [comment]
Yes it's a hosted github enterprise instance.  I'll have to remake my changes (I tore it up).

**Comment by ggb667** [comment]
```
c:\wk\ms-attraction-reservations>"C:\Program Files\nodejs\node.exe" c:\wk\pullr -g https://myservername/directory-name -n -l -t "Sonar Jenkins Update" -d "Update Jenkins file so it supports develop branch as opposed to all branches together" -I "develop" -f origin/feature/US32497-1_Sonar -p --plaintext
 Please Enter Your GitHub Credentails
 Email: gregory.bishop@univeralorlando.com
 Password:
 Error: Invalid GitHub Credentials
```

`git config -l` shows:

user.name=Gregory Bishop
user.email=gregory.bishop@univeralorlando.com
core.askpass=
http.sslverify=false
credential.https://usolxpgithub01.use.ucdp.net.username=206564814
credential.https://usolxpgithub01.use.ucdp.net.helper=wincred
http.sslcainfo=C:/Program Files/Git/mingw64/ssl/certs/ca-bundle.crt
http.sslbackend=schannel

Also tried without SSL, but no dice there also.
